core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -987,6 +977,7 @@ public void bufferUpdate(BufferUpdateEvent event) {
     @Subscribe(threadMode = ThreadMode.MAIN)
     @SuppressWarnings("unused")
     public void sleepTimerUpdate(SleepTimerUpdatedEvent event) {
+        updateMediaSession(mediaPlayer.getPlayerStatus());
```

Comment:
Hmm, this would update the media session every single second. I would rather check `event.wasJustEnabled` and `event.isOver` or `event.isCancelled`